packages/angular/src/builders/delegate-build/schema.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface DelegateBuildExecutorSchema {
+  buildTarget: string;
+  outputPath: string;
+  tsConfig: string;
```

Comment:
I understand that `tsConfig` is required for the Incremental Build to work. Why do we need `outputPath`? should that not be defined in the `buildTarget`'s options?

Reply:
That's required by the `@nrwl/web:file-server` to serve the built output.

Reply:
Is it something that can be defined in the `serve` target?

Reply:
It's a good point. As discussed offline, we'll leave this for a future PR if needed. Changing the `@nrwl/web:file-server` schema can impact the usage in other frameworks as well. Also, most (if not all) of the custom executors people use today for building Angular apps, are based or delegate the build to the `@angular-devkit/build-angular:browser` which does support that property.